compiler/noirc_frontend/src/hir/comptime/interpreter.rs

```diff
@@ -131,7 +145,7 @@ impl<'local, 'interner> Interpreter<'local, 'interner> {
         Ok(result)
     }
 
-    fn call_builtin(
+    fn call_special(
```

Comment:
Why the rename?

Reply:
The `call_builtin` in builtin.rs is now a method on the interpreter since it now needs an interpreter parameter.
This function dispatches to builtins, lowlevel, and foreign functions so I figured this was the better candidate to rename.